src/xcode/ENA/ENA/Source/Extensions/FileManager+ClearTemporaryDirectory.swift

```diff
@@ -0,0 +1,22 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+ extension FileManager {
+	func removePDFsFromTemporaryDirectory() {
```

Comment:
Optional:
We could handle that stuff on a background thread, because it's independent from the sync stuff on main thread.

```swift
DispatchQueue.global(qos: .background).async { ... }
```